### PR TITLE
[FB-328] Add new hero

### PR DESCRIPTION
**Pull request by Roller23**
Adds the new Hero

Screenshot 2023-07-24 at 20 24 32

### CODE REVIEW

File: src/components/CustomBulletList/CustomBulletList.tsx [hunk 1]
```diff
@@ -15,25 +14,20 @@ const List = styled.ul`
...
-`;
-
-const StyledTickBlue = styled(TickBlue)`
-  content: '';
-  position: absolute;
-  left: 0;
-  width: 36px;
-  height: 36px;
+const CheckmarkWrapper = styled.div`
+  color: ${({ color = 'black' }) => color};
```
Comment (kamilKamilson) [inline src/components/CustomBulletList/CustomBulletList.tsx:18]: nit: is it necessary? you are not passing prop color to this styled component during render

File: src/sections/Hero/Hero.tsx [hunk 2]
```diff
@@ -88,28 +86,23 @@ export const Hero = () => {
 
   return (
     <FeaturedCase>
-      <h1 style={{ color: 'white', fontWeight: 600 }}>Forminer</h1>
-      <StyledLargeText>
-        <strong>Save up to 1500 hours</strong> while building form-heavy React
-        apps with our easy to integrate package
-      </StyledLargeText>
+      <h2 style={{ fontWeight: 900 }}>
```
Comment (kamilKamilson) [inline src/sections/Hero/Hero.tsx:89]: why there is h2 instead of h1?

File: src/sections/TrustedBy/TrustedBy.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    padding: 0px;
+  `}
+`;
+
+const CompanyImage = styled.img`
+  height: 100%;
+`;
+
+export const TrustedBy = () => {
+  return (
```
Comment (kamilKamilson) [inline src/sections/TrustedBy/TrustedBy.tsx:50]: nit: unnecessary return

**Review by kamilKamilson — CHANGES_REQUESTED** [review]

**Comment by kamilKamilson** [comment]
what should be improved:

- hover animation on hero CTA buttons, they shouldnt be this simmilar
https://github.com/vazco/forminer/assets/49162755/998156b5-e92c-449c-9816-f5cb71f08256

- hover on navbar
image

- responsiveness on small devices (iPhone SE)
image

-maybe change this to auto slides?
image

**Review by kamilKamilson — APPROVED** [review]

File: src/sections/Hero/Hero.tsx (hunk 2 above)
Comment (Roller23) [reply src/sections/Hero/Hero.tsx:89]: H1 seemed a bit too big

**Pull request merged**